client/components/keyed-suggestions/index.jsx

```diff
@@ -172,8 +174,19 @@ class KeyedSuggestions extends Component {
 	onMouseDown = ( event ) => {
 		event.stopPropagation();
 		event.preventDefault();
+
+		const { suggest, recordTracksEvent } = this.props;
 		const suggestion = event.currentTarget.textContent.split( ' ' )[ 0 ];
-		this.props.suggest( suggestion, this.state.isShowTopLevelTerms );
+		if ( this.state.isShowTopLevelTerms ) {
+			// Clean up suggestion that it's passed as, for example, "feature" instead of "feature:".
+			recordTracksEvent( 'search_dropdown_searchby_item_click', {
+				term: suggestion.split( ':' )[ 0 ],
+			} );
+		} else {
+			recordTracksEvent( 'search_dropdown_taxonomy_item_click', { term: suggestion } );
```

Comment:
Similar to the previous comment, what about `search_dropdown_taxonomy_term_click`?